lib/shipit/task_commands.rb

```diff
@@ -41,6 +41,8 @@ def env
         'IGNORED_SAFETIES' => @task.ignored_safeties? ? '1' : '0',
         'GIT_COMMITTER_NAME' => @task.user&.name || Shipit.committer_name,
         'GIT_COMMITTER_EMAIL' => @task.user&.email || Shipit.committer_email,
+        'GITHUB_REPO_OWNER' => @stack.repository.owner,
+        'GITHUB_REPO_NAME' => @stack.repository.name,
```

Comment:
Hum, I wonder if it should be two variables, or just one with the full name.

Reply:
It's easy enough for users to compose if they choose, rather than having to figure out some bash foo to split them.

Reply:
We could do both of course, and add a `GITHUB_REPO_FULL_NAME` or something.